src/utils/helpers.h

```diff
@@ -515,7 +515,7 @@ inline Solution format_solution(const Input& input,
                  [&](auto j) { return input.jobs[j]; });
 
   return Solution(0,
-                  input.zero_amount().size(),
+                  input.zero_amount(),
```

Comment:
A similar change is missing below in `format_solution` for TW routes (L928). I'm actually a bit surprised that passing an `unsigned` as `const Amount&` over there even compiles.

Reply:
Thank you for the review.

Yes, I realised I can do some more changes and will be back with them.

Reply:
> I'm actually a bit surprised that passing an `unsigned` as `const Amount&` over there even compiles.

That I think is due to non-explicit `Amount(std::size_t size)` ctor allowing implicit conversion.